common/src/main/java/com/wynntils/core/consumers/features/Feature.java

```diff
@@ -41,7 +41,7 @@ public String getTranslatedDescription() {
     }
 
     public String getShortName() {
```

Comment:
Are you sure that changing this does not have any unwanted side effects?

Reply:
I checked that a while ago and as far as I can see this method (getShortName) is only used by getFeatureByName (or getOverlayByName) and that is only used by commands but someone should definitely verify that.

Reply:
Yes, the only places this function is used are for /config and /feature and for logging.